src/graphql/data/SearchTokens.ts

```diff
@@ -0,0 +1,97 @@
+import { WRAPPED_NATIVE_CURRENCY } from 'constants/tokens'
+import gql from 'graphql-tag'
+import { useMemo } from 'react'
+
+import { Chain, SearchTokensQuery, useSearchTokensQuery } from './__generated__/types-and-hooks'
+import { chainIdToBackendName } from './util'
+
+gql`
+  query SearchTokens($searchQuery: String!) {
+    searchTokens(searchQuery: $searchQuery) {
```

Comment:
Is there a way we can modify the ordering? I believe the old api was based off volume but for example searching `eth` steth didn't show up for me which I would have expected.
new results:
![Screen Shot 2023-01-10 at 09 08 29 ](https://user-images.githubusercontent.com/11512321/211618926-4460cf17-5165-404f-9062-677fcf419315.png)
old resultss:
![Screen Shot 2023-01-10 at 09 08 36 ](https://user-images.githubusercontent.com/11512321/211618994-eda9385a-bc60-4a7c-93d0-668ffae8028e.png)

Reply:
This might actually be a data issue, for some reason stEth is only returning as being on Polygon

Reply:
I actually noticed this same issue! Matteen is looking into it, seems like a data issue. They are sorted by volume

Reply:
this is fixed now

Reply:
Ordering still seems funky
<img width="539" alt="Screen Shot 2023-02-05 at 20 31 26 " src="https://user-images.githubusercontent.com/11512321/216884190-09fb750c-5ab5-4260-97b9-c147942dc769.png">